### PR TITLE
Connecting appellant/veteran information to backend

### PR DESCRIPTION
**Pull request by sharonwarner**
Connects #2964 

- [ ] All appellant/veteran information is now pulled from the backend, except docket number (that's being moved into the appeal section in a future card

screen shot 2017-09-13 at 4 44 14 pm

### CODE REVIEW

File: app/models/hearing.rb [hunk 1]
```diff
@@ -90,7 +90,10 @@ def to_hash
 
   def to_hash_with_appeals_and_issues
```
Comment (aroltsch) [inline app/models/hearing.rb]: I think we should re-name this method as it contains more info now.

File: app/models/hearing.rb [hunk 2]
```diff
@@ -51,14 +51,15 @@ def request_type
...
-    :veteran_full_name, \
+    :veteran_name, \
     :appellant_last_first_mi, \
     :appellant_city, \
     :appellant_state, \
     :regional_office_name, \
     :vbms_id, \
     :number_of_documents, \
     :number_of_documents_after_certification, \
+    :representative, \
```
Comment (aroltsch) [inline app/models/hearing.rb]: What is the difference between representative (that we get from `brieff.bfso`) and representative_name (that we get from `hearshed.repname`)?

File: app/models/hearing.rb [hunk 3]
```diff
@@ -76,9 +77,8 @@ def to_hash
         :appellant_last_first_mi,
         :appellant_city,
         :appellant_state,
-        :representative_name,
```
Comment (aroltsch) [inline app/models/hearing.rb]: Are we not using hearshed.repname anymore?

File: app/models/hearing.rb (hunk 2 above)
Comment (sharonwarner) [reply app/models/hearing.rb]: Yeah, there are two fields on the hearing worksheet for representative. There's representative_name which is from hearsched, and then there's representative which comes from the appeal in brieff.bfso. This ticket only links the appeal's representative info up.

File: app/models/hearing.rb (hunk 3 above)
Comment (sharonwarner) [reply app/models/hearing.rb]: Good call, I meant to move it to the worksheet's to_hash function, but I can do that in a later ticket.

File: app/models/hearing.rb (hunk 1 above)
Comment (sharonwarner) [reply app/models/hearing.rb]: 👍

**Pull request merged**